cloudera/assets/configuration/spec.yaml

```diff
@@ -132,4 +132,12 @@ files:
               type: array
               items:
                 type: string
+    - name: cloudera_client
```

Comment:
It is possible that not, but I prefer to have an external configuration option in case in the future the client used is another, new versions, change the name, etc... than to have it set in stone in our code.